cloud/src/common/bvars.cpp

```diff
@@ -186,4 +188,180 @@ BvarStatusWithTag<int64_t> g_bvar_inverted_checker_abnormal_delete_bitmaps("chec
 BvarStatusWithTag<int64_t> g_bvar_inverted_checker_delete_bitmaps_scanned("checker", "delete_bitmap_keys_scanned");
 BvarStatusWithTag<int64_t> g_bvar_max_rowsets_with_useless_delete_bitmap_version("checker", "max_rowsets_with_useless_delete_bitmap_version");
 
+// rpc kv rw count
```

Comment:
seems lack of instance level stats
we may need to add `instance_id` a mbvar label.